rust/publish-ghost/src/lib.rs

```diff
@@ -15,12 +128,18 @@ pub struct Cli {
     path: PathBuf,
 
     /// The Ghost domain
-    #[arg(long, env = "STENCILA_GHOST_DOMAIN")]
-    domain: bool,
+    /// 
+    /// This is the domain name of your Ghost instance, with an optional port. 
+    #[arg(long, env = "STENCILA_GHOST_DOMAIN", value_parser = parse_host)]
+    ghost: Host,
 
     /// The Ghost Admin API key
-    #[arg(long, env = "STENCILA_GHOST_KEY")]
-    key: bool,
+    /// 
+    /// To create one, create a new Custom Integration under
+    /// the Integrations screen in Ghost Admin. Use the Admin API Key,
+    /// rather than the Content API Key.
+    #[arg(long, env = "STENCILA_GHOST_KEY", value_parser = parse_key)]
```

Comment:
Mention in docs that can set the secret to avoid entering it each time:

```sh
stencila secrets set GHOST_ADMIN_API_KEY
```

Reply:
Also should have better consistency between the secret name and the env var name: maybe the secret name should be the env var name minus the `STENCILA_` prefix